### ISSUE TITLE
Compress option does not work

### ISSUE THREAD

**Issue by davisarchibald** [MadebyAe/node-handlebars#2]
Currently the output is automatically minimized but according to the documentation the output should not be automatically compressed. When trying to manually set compress option to false, still didn't work. Found the issue was that compress is no longer used in the code, and instead minimize is being used to compress the code. I'll be adding a pull request to update documentation so that further users can avoid this confusion.

**Comment by davisarchibald** [comment]
Pull request here: https://github.com/MadebyAe/node-handlebars/pull/3

**Comment by shaoyihe** [comment]
Yes , it was. Now it's still wrong .You can directly use `handlebars` .eg:

```
var hbs = require("handlebars").create();
var hrml = hbs.compile(fs.readFileSync(path.resolve(__dirname, "sample.json")).toString())(json);
```

**Comment by MadebyAe** [comment]
Thanks we are working on v1.0 new features coming :)
